bmaptools/BmapHelpers.py

```diff
@@ -61,7 +61,15 @@ def get_block_size(file_obj):
     # Get the block size of the host file-system for the image file by calling
     # the FIGETBSZ ioctl (number 2).
     binary_data = ioctl(file_obj, 2, struct.pack('I', 0))
-    return struct.unpack('I', binary_data)[0]
+    bsize = struct.unpack('I', binary_data)[0]
+    if not bsize:
+        import os
```

Comment:
It would be good to be consistent about where standard modules are imported. I think most of the code imports everything at the beginning, and there may be some exceptions when it comes to some rare/special modules, which are imported inside functions. But stuff like 'os' would always be imported at the beginning. I personally think it is better this way.

But if you prefer "on-demand" import, it is better to change it globablly, rather than doing leaving the code inconsistent.